src/main/java/components/Storage.java

```diff
@@ -0,0 +1,88 @@
+package components;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents storage, which stores user's inputted tasks in a text file.
+ */
+public class Storage {
+
+  private File f;
+  private ArrayList<Task> ls;
```

Comment:
Perhaps we could change the name of ls to a pural form of collection of objects.

Reply:
ditto comment, actualy nouns instead of shortforms would probably be useful too!